content/flux/v0.x/data-types/composite/record.md

```diff
@@ -10,4 +10,168 @@ menu:
     parent: Composite types
 weight: 201
 flux/v0.x/tags: ["composite types", "data types"]
----
\ No newline at end of file
+---
+
+A **record** type is a set of key-value pairs (also known as properties).
```

Comment:
Should we define properties in our glossary (are we using this term throughout the docs in the same way?). 
Noticed our definition of field refers to "the key-value pair in InfluxDB’s data structure that records metadata and the actual data value." 

Would it be accurate to say a record is a set of fields (or tags?) also known as properties? If not, maybe helpful to clarify this distinction as appropriate throughout.

Reply:
No, the term "properties" is specific to records. The terms "fields" or "tags" don't really have meaning within the Flux language. They are concepts tied to the InfluxDB storage engine.